SingularityUI/app/components/common/modal/FormModal.jsx

```diff
@@ -142,11 +141,11 @@ export default class FormModal extends React.Component {
     if (event) {
       event.preventDefault();
     }
-    if (this.validateForm()) {
+    if (this.isValidForm()) {
       let formState = this.parseFormState(this.state.formState);
       this.props.onConfirm(formState);
       if (!this.props.keepCurrentFormState) {
-        const formState = {};
+        formState = {};
```

Comment:
removed the const because the variable is block scoped and so it would never be saved properly here